src/integrationTest/java/org/openstreetmap/atlas/generator/AtlasGeneratorIntegrationTest.java

```diff
@@ -51,7 +54,14 @@
     public static final String SLICING_CONFIGURATION = CONFIGURATION
             + "/atlas-relation-slicing.json";
 
-    static
+    @After
+    public void clean()
+    {
+        ResourceFileSystem.clear();
```

Comment:
hmm what does this do?

Reply:
The ResourceFileSystem is a static hash map of string path to resource objects. It implements the minimum possible of the FileSystem API using the backing of this hash map.  This method clears the static map, so as to look like the file system is empty.
That clean here is also the reason I moved the filling of the file system in the `@Before` method so that it is ready to use before each test

Reply:
The reason why I have done that is I had some random test errors locally, which looked like they came from the fact that the FileSystem was still containing the output of one test when the other test started, which prompted it to fail.